upgrade/scripts/k8s/upgrade_k8s_steps.sh

```diff
@@ -119,5 +119,18 @@ else
   fi
 fi
 
+if [[ -f "$DONE_DIR/rr_cs_rollout_restart.done" ]]; then
```

Comment:
Adding this here will address the CSM upgrade case, but this also needs to happen on CSM fresh installs.